timesketch/frontend/src/views/SketchExplore.vue

```diff
@@ -361,7 +361,7 @@ const defaultQueryFilter = () => {
     'time_end': null,
```

Comment:
can you remove `time_start` and `time_end` since this has been removed from teh default in other places (and no longer a thing int he `build_query`)

Reply:
Good catch, removed now.